diff --git format_checker/tool/rustfmt_with_policies.sh format_checker/tool/rustfmt_with_policies.sh
index 87d356d..1beb693 100755
--- format_checker/tool/rustfmt_with_policies.sh
+++ format_checker/tool/rustfmt_with_policies.sh
@@ -24,7 +24,7 @@ source "${RUNFILES_DIR:-/dev/null}/$f" 2>/dev/null || \
   { echo>&2 "ERROR: cannot find $f"; exit 1; }; f=; set -e
 # --- end runfiles.bash initialization v3 ---
 
-RUSTFMT_BIN="$(rlocation rules_rust/tools/upstream_wrapper/rustfmt)"
+RUSTFMT_BIN="$(rlocation score_rules_rust/tools/upstream_wrapper/rustfmt)"
 RUSTFMT_CONFIG="$(rlocation score_rust_policies/rustfmt/rustfmt.toml)"
 
 exec "${RUSTFMT_BIN}" --config-path "${RUSTFMT_CONFIG}" "$@"
